client/stores/addon.js

```diff
@@ -0,0 +1,30 @@
+import { observable, computed, action, runInAction } from 'mobx'
+import axios from 'axios'
+
+class AddonStore {
+	constructor () {
+		// Fetch list of addons from website server
+		this.getAddons()
+	}
+
+	@observable addons = []
+
+	@computed get addonsSortedByMostRecent () {
+		return this.addons.sort((a, b) => {
+			if (a.lastUpdated > b.lastUpdated) return -1
+			else if (a.lastUpdated < b.lastUpdated) return 1
+
+			return 0
+		})
+	}
+
+	@action getAddons = async () => {
+		const addons = await axios.get('/api/addons').then(res => res.data.addons)
+
+		runInAction(() => {
+			this.addons = addons
+		})
+	}
+}
+
+export default new AddonStore()
```

Comment:
Yeah I went back and forth on this. I've seen collections referred to with either naming convention, so I just picked one. Looking at it now I agree with your reasoning for the plural and will move to that